packages/react-components/src/components/VideoGallery/utils/videoGalleryLayoutUtils.ts

```diff
@@ -32,42 +32,46 @@ const DEFAULT_MAX_REMOTE_VIDEOSTREAMS = 4;
 const DEFAULT_MAX_AUDIO_DOMINANT_SPEAKERS = 6;
 
 const _useOrganizedParticipants = (props: OrganizedParticipantsArgs): OrganizedParticipantsResult => {
-  const visibleVideoParticipants = useRef<VideoGalleryRemoteParticipant[]>([]);
-  const visibleAudioParticipants = useRef<VideoGalleryRemoteParticipant[]>([]);
+  const visibleGridParticipants = useRef<VideoGalleryRemoteParticipant[]>([]);
+  const visibleHorizontalGalleryParticipants = useRef<VideoGalleryRemoteParticipant[]>([]);
 
   const {
-    remoteParticipants,
+    remoteParticipants = [],
     dominantSpeakers,
     maxRemoteVideoStreams = DEFAULT_MAX_REMOTE_VIDEOSTREAMS,
-    maxAudioDominantSpeakers = DEFAULT_MAX_AUDIO_DOMINANT_SPEAKERS,
-    isScreenShareActive = false
+    maxHorizontalGalleryDominantSpeakers: maxAudioDominantSpeakers = DEFAULT_MAX_AUDIO_DOMINANT_SPEAKERS,
```

Comment:
should this be `maxOverflowGalleryDominantSpeakers`?

Reply:
Yup. Thanks.